app/src/main/java/com/epfl/drawyourpath/challenge/milestone/Milestone.kt

```diff
@@ -0,0 +1,24 @@
+package com.epfl.drawyourpath.challenge.milestone
+
+import com.epfl.drawyourpath.utils.Utils
+import java.time.LocalDate
+
+data class Milestone(
+    val name: String,
+    val description: String,
+    val date: LocalDate,
+    val drawable: Int,
+    val dateAsString: String = Utils.getDateAsString(date),

```

Comment:
Maybe you can add a default value for the drawable, like you have done for the trophy.